reframe/frontend/statistics.py

```diff
@@ -129,9 +129,9 @@ def performance_report(self):
 
             for key, ref in t.check.perfvalues.items():
                 var = key.split(':')[-1]
-                val = ref[0]
+                val = ref[1]
```

Comment:
I am wondering what has changed in the reference tuple?

Reply:
It is changed here: https://github.com/eth-cscs/reframe/blob/4d17759f5097b5ab65607a38b0f2be0e8095bc9b/reframe/core/pipeline.py#L1137

Reply:
This change in `pipeline.py` is redundant. I suggest reverting it back without the `tag`. Then `statistics` will be fine again.